schema/deploy/warehouse/site/data.sql

```diff
@@ -35,6 +38,10 @@ insert into warehouse.site(identifier, details)
             "sample_origin": "kp"}'),
         ('KingStreetStation',                       '{"category": "community",  "type": "publicSpace",
             "swab_site": "king_street%"}'),
+        ('MightyKidz',                              '{"category": "community",  "type": "childcare"}'),
+        ('MinorAvenueChildrensHouse',               '{"category": "community",  "type": "childcare"}'),
+        ('MothersPlace',                            '{"category": "community",  "type": "childcare"}'),
+        ('PICAWA',                                  '{"category": "community",  "type": "SCAN"}'),
```

Comment:
I believe this site was in-person collections at PICAWA, not the mail-in type samples. Should the `type` be "publicSpace"?

Reply:
It was in-person, yes. I wasn't sure what `type` should be here since my understanding (from Slack logs :no_mouth:) was that PICAWA was part of the SCAN outreach, and when I saw that the `SCAN` "site" also had a `SCAN` type, I thought maybe PICAWA should match.

Reply:
Ah right, this goes back to our Slack [convo](https://seattle-flu-study.slack.com/archives/C0173B4S1LY/p1625251229396900) of what should `site` represent. Should we continue to follow the model of physical collection sites/kiosks get a true `type` while mail-in sites have their own `type`?

Reply:
Yeah, I think that makes more sense upon further consideration. Repushed!